source/_integrations/history_stats.markdown

```diff
@@ -178,6 +178,18 @@ Here, last Monday is _today_ as a timestamp, minus 86400 times the current weekd
 
 {% endraw %}
 
+**Next 4pm **: from 4pm today until 4pm tomorrow.
+
+{% raw %}
+
+```yaml
+    end: '{{ (now().replace(minute=0,second=0) + timedelta(hours=8)).replace(hour=16) }}'
```

Comment:
So why the juggling with all the hours? And not get today's 4pm + 1 day?

`{{ (now().replace(hour=16, minute=0,second=0) + timedelta(days=1)) }}`

Is there a specific reason for your approach?